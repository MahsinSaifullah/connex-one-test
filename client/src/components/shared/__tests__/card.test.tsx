import { render } from '@testing-library/react';

import { Card } from '../card.component';

const TestCard = () => {
  return (
    <Card>
      <div>Test</div>
    </Card>
  );
};

describe('Title Component', () => {
  it('should render title component correctly', () => {
    const { asFragment } = render(<TestCard />);
    expect(asFragment()).toMatchSnapshot();
  });
});
