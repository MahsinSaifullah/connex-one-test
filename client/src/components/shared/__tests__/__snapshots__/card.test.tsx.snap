// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component should render title component correctly 1`] = `
<DocumentFragment>
  <article
    class="rounded shadow-lg bg-white overflow-scroll undefined"
  >
    <div>
      Test
    </div>
  </article>
</DocumentFragment>
`;
